### ISSUE TITLE
not enough logs

### ISSUE THREAD

**Issue by yegor256** [objectionary/opeo-maven-plugin#33]
I'm getting this:

```
[INFO] --- opeo:0.0.2:decompile (default-cli) @ benchmark ---
[INFO] opeo-maven-plugin: started decompiling bytecode into EO
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS

[INFO] --- opeo:0.0.2:compile (default-cli) @ benchmark ---
[INFO] opeo-maven-plugin: started compiling EO into low-level representation
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
```

This is not enough. Let's say something like this:

```
[INFO] --- opeo:0.0.2:decompile (default-cli) @ benchmark ---
[INFO] Decompiling EO sources from target/eo
[INFO] Saving new EO sources to target/eo-2
[INFO] Decompiled app.eo (545 bytes)
[INFO] Decompiled main.eo (545 bytes)
[INFO] Totally decompiled 2 EO sources
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS

[INFO] --- opeo:0.0.2:compile (default-cli) @ benchmark ---
[INFO] Compiling EO sources from target/eo
[INFO] Saving new EO sources to target/eo-2
[INFO] Compiled app.eo (545 bytes)
[INFO] Compiled main.eo (545 bytes)
[INFO] Totally compiled 2 EO sources
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
```

**Comment by yegor256** [comment]
@volodya-lombrozo please, help

**Comment by volodya-lombrozo** [comment]
@yegor256 Ok I will do it

**Comment by yegor256** [comment]
@volodya-lombrozo for reporting the size of the files you can use `%[size]s` decor in jcabi `Logger`

**Comment by volodya-lombrozo** [comment]
@yegor256 Do you have a link to the doc? is `size` should be an int value?

**Comment by yegor256** [comment]
@volodya-lombrozo see this README: https://github.com/jcabi/jcabi-log (yes, size can be long or int)

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38) are still not solved.

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37) is still not solved; solved: [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38).

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62) is still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38).

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65).

**Comment by 0pdd** [comment]
@yegor256 6 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77).

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78).

**Comment by 0pdd** [comment]
@yegor256 6 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78).

**Comment by 0pdd** [comment]
@yegor256 6 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87), [#89](https://github.com/objectionary/opeo-maven-plugin/issues/89) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86).

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86), [#89](https://github.com/objectionary/opeo-maven-plugin/issues/89).

**Comment by 0pdd** [comment]
@yegor256 6 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87), [#97](https://github.com/objectionary/opeo-maven-plugin/issues/97), [#98](https://github.com/objectionary/opeo-maven-plugin/issues/98) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86), [#89](https://github.com/objectionary/opeo-maven-plugin/issues/89).

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#134](https://github.com/objectionary/opeo-maven-plugin/issues/134), [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87), [#97](https://github.com/objectionary/opeo-maven-plugin/issues/97) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86), [#89](https://github.com/objectionary/opeo-maven-plugin/issues/89), [#98](https://github.com/objectionary/opeo-maven-plugin/issues/98).

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#134](https://github.com/objectionary/opeo-maven-plugin/issues/134), [#153](https://github.com/objectionary/opeo-maven-plugin/issues/153), [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87) are still not solved; solved: [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86), [#89](https://github.com/objectionary/opeo-maven-plugin/issues/89), [#97](https://github.com/objectionary/opeo-maven-plugin/issues/97), [#98](https://github.com/objectionary/opeo-maven-plugin/issues/98).

**Comment by 0pdd** [comment]
@yegor256 4 puzzles [#134](https://github.com/objectionary/opeo-maven-plugin/issues/134), [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87) are still not solved; solved: [#153](https://github.com/objectionary/opeo-maven-plugin/issues/153), [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86), [#89](https://github.com/objectionary/opeo-maven-plugin/issues/89), [#97](https://github.com/objectionary/opeo-maven-plugin/issues/97), [#98](https://github.com/objectionary/opeo-maven-plugin/issues/98).

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#74](https://github.com/objectionary/opeo-maven-plugin/issues/74), [#75](https://github.com/objectionary/opeo-maven-plugin/issues/75) are still not solved; solved: [#134](https://github.com/objectionary/opeo-maven-plugin/issues/134), [#153](https://github.com/objectionary/opeo-maven-plugin/issues/153), [#37](https://github.com/objectionary/opeo-maven-plugin/issues/37), [#38](https://github.com/objectionary/opeo-maven-plugin/issues/38), [#62](https://github.com/objectionary/opeo-maven-plugin/issues/62), [#65](https://github.com/objectionary/opeo-maven-plugin/issues/65), [#76](https://github.com/objectionary/opeo-maven-plugin/issues/76), [#77](https://github.com/objectionary/opeo-maven-plugin/issues/77), [#78](https://github.com/objectionary/opeo-maven-plugin/issues/78), [#80](https://github.com/objectionary/opeo-maven-plugin/issues/80), [#81](https://github.com/objectionary/opeo-maven-plugin/issues/81), [#86](https://github.com/objectionary/opeo-maven-plugin/issues/86), [#87](https://github.com/objectionary/opeo-maven-plugin/issues/87), [#89](https://github.com/objectionary/opeo-maven-plugin/issues/89), [#97](https://github.com/objectionary/opeo-maven-plugin/issues/97), [#98](https://github.com/objectionary/opeo-maven-plugin/issues/98).
